response_operations_ui/controllers/collection_exercise_controllers.py

```diff
@@ -161,6 +161,26 @@ def get_case_group_status_by_collection_exercise(case_groups, collection_exercis
                 if case_group['collectionExerciseId'] == collection_exercise_id)
 
 
+def unlink_sample_summary(collection_exercise_id, sample_summary_id):
+    logger.debug('un-linking sample summary from collection exercise', collection_exercise_id=collection_exercise_id,
+                 sample_summary_id=sample_summary_id)
+    url = f'{app.config["COLLECTION_EXERCISE_URL"]}/collectionexercises/unlink/{collection_exercise_id}' \
+          f'/sample/{sample_summary_id}'
+
+    response = requests.delete(url, auth=app.config['COLLECTION_EXERCISE_AUTH'])
+
+    try:
+        response.raise_for_status()
+    except HTTPError:
+        logger.exception('Failed to unlink sample summary from collection exercise',
+                         collection_exercise_id=collection_exercise_id, sample_summary_id=sample_summary_id)
+        return False
+
+    logger.debug('Successfully unlinked sample summary from a collection exercise',
+                 collection_exercise_id=collection_exercise_id, sample_summary_id=sample_summary_id)
+    return response.json
```

Comment:
Should this be `response.json()`?

Reply:
Request to delete sample link does not return a JSON response. Maybe just return True?

Reply:
No it doesn't return any response probably best just to return true